src/pages/EachCategory.js

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { nobelPrizeActions } from '../redux/nobel-prize';
+
+const EachCategory = () => {
+  const dispatch = useDispatch();
+  const params = useParams();
+  const nobelPrizes = useSelector((state) => state.nobel);
+  const newArray = [];
+  for (let i = 0; i < nobelPrizes.length; i += 1) {
+    if (nobelPrizes[i].category === params.categoryName) {
+      newArray.push(nobelPrizes[i]);
+    }
+  }
+  const [searchTerm, setSearchTerm] = useState([]);
+  const handleChange = (event) => {
+    setSearchTerm(event.target.value);
+  };
+  const filteredNewArray = newArray.filter((item) => item.year.toLowerCase().includes(searchTerm));
```

Comment:
- [ ] Great job working on this capstone project. Your category page is not displaying the list of noble price winners. When i try refreshing the page after clicking on a category, I get and error. See both screenshots below.
- **No list of noble prize winners diplaying for the medcine catgory**
![Screen Shot 2023-02-15 at 4 47 27 AM](https://user-images.githubusercontent.com/62849872/219035376-ff1b306d-3cbf-42b6-ab32-78483d1f10ba.png)
- **Error when I refresh the page**
![Screen Shot 2023-02-15 at 4 45 02 AM](https://user-images.githubusercontent.com/62849872/219035669-20eb7863-9c62-4734-bc79-c7cae83e0854.png)

- Check the console as I am getting an error with the API GET method for fectching data for categories.
![Screen Shot 2023-02-15 at 5 24 45 AM](https://user-images.githubusercontent.com/62849872/219039790-83e4cfdc-efeb-452f-ba8b-37bb78ec8d0b.png)

Reply:
Hi @t-yanick, I can checked it from my end with a different device and it is working pretty well. 
https://user-images.githubusercontent.com/110017001/219054667-c7126a6f-3f23-41b6-92e5-f4ce95e7be62.MOV

Reply:
- Kindly fix all the other issues and request for a re-review.